src/content/docs/apm/agents/c-sdk/instrumentation/distributed-tracing-c-sdk.mdx

```diff
@@ -0,0 +1,214 @@
+---
+title: 'Distributed tracing for your C services'
+tags:
+  - Agents
+  - C SDK
+  - Instrumentation
+metaDescription: 'For the New Relic C SDK: how to implement distributed tracing across your C language applications and services.'
+redirects:
+  - /docs/agents/c-sdk/instrumentation/enable-distributed-tracing-your-c-applications
+  - /docs/apm/agents/c-sdk/instrumentation/enable-distributed-tracing-your-c-applications
+---
+
+import logsInContextTraces from 'images/logs-in-context-traces.png'
+
+import clogo from 'images/clogo.png'
+
+<Callout
+  variant="important"
+  title="EOL NOTICE"
+  >
+  As of April 2022, we're discontinuing support for several capabilities, including the C SDK. For more details, including how you can easily prepare for this transition, see our [Explorers Hub post](https://discuss.newrelic.com/t/q1-bulk-eol-announcement-fy23/181744).
+</Callout>
+
+Distributed tracing allows you to see the entire journey of your requests throughout a [distributed system](/docs/distributed-tracing/concepts/introduction-distributed-tracing). The [C SDK](https://github.com/newrelic/c-sdk) supports [standard distributed tracing](/docs/distributed-tracing/concepts/how-new-relic-distributed-tracing-works/#head-based) with head-based sampling. It doesn't support the feature called Infinite Tracing or W3C Trace context.
+
+The C SDK requires you to manually instrument your C services, unlike the auto-instrumentation of other New Relic agents. This means you need to add some lines to your code to use the C SDK, and then you add an additional configuration to enable distributed tracing.
+
+Here's how to get started:
+
+* [Instrument a sample program (recommended):](#quick-start-apm) A fast way to get started
+* [Manual instrumentation:](#manual-instrumentation) Tips for additional instrumentation
```

Comment:
Remove this link

Reply:
Good catch! I replaced this bullet with this:

* [Instrument transactions and HTTP requests](#guidelines)